examples/learning/learn_associations.ipynb

```diff
@@ -0,0 +1,330 @@
+{
+ "metadata": {
+  "kernelspec": {
```

Comment:
I used `ipython nbconvert --to notebook --nbformat 3 examples/learning/learn_associations.ipynb` precisely because I am using IPython 3 (nbformat 4), while our other notebooks are in IPython 2 (nbformat 3). If you look at line 22, it should be the correct format and version. See https://github.com/ipython/ipython/issues/8075. Thinking it's fine (just extra metadata)... In the future we should probably think about running this command automatically as a hook.